src/compass/utils/lut.py

```diff
@@ -7,23 +19,333 @@ def compute_geocoding_correction_luts(burst, rg_step=200, az_step=0.25):
     ----------
     burst: Sentinel1BurstSlc
         S1-A/B burst object
+    cfg: dict
+        Dictionary containing user processing parameters
+    geogrid: isce3.product.geogrid
+        ISCE3 geogrid object with user-defined inputs
     xstep: int
         LUT spacing (in pixels) along x/slant range in meters
     ystep: int
         LUT spacing (in pixels) along x/azimuth in seconds
 
     Returns
     -------
-    rg_lut: ndarray
-        2D array containing sum of range corrections
-    az_lut: ndarray
-        2D array containg sum of azimuth corrections
+    rg_lut: isce3.core.LUT2d
+        Sum of range corrections in meters
+    az_lut: isce3.core.LUT2d
+        Sum of azimuth corrections in meters
     '''
 
-    az_bistatic = burst.bistatic_delay(range_step=rg_step, az_step=az_step)
-    rg_doppler = burst.geometrical_and_steering_doppler(range_step=rg_step, az_step=az_step)
+    # Get some ancillary parameters
+    dem_raster = isce3.io.Raster(cfg.dem)
+    date_str = burst.sensing_start.strftime("%Y%m%d")
+    scratch_path = f'{cfg.scratch_path}/{burst.burst_id}/{date_str}'
```

Comment:
```suggestion
    scratch_path = cfg.output_paths.scratch_path
```
Utilize [paths](https://github.com/opera-adt/COMPASS/blob/b5e9341fdb3a1cd9a624a24410b1b05a215c48da/src/compass/utils/runconfig.py#L296) stored in from `runconfig.output_paths`.